package.json

```diff
@@ -43,7 +44,7 @@
     "moment": "^2.29.1",
     "moment-timezone": "^0.5.32",
     "qs": "^6.9.6",
-    "rxjs": "^6.6.3"
+    "rxjs": "^6.2.1"
```

Comment:
I think we should be able to remove `rxjs` from our internal dependency list because we have it as a peer dep.

Reply:
Done ✅